### PR TITLE
Multiplication table in python

### PR DESCRIPTION
**Pull request by RashiniKaweesha**
created multiplication table iv python

### CODE REVIEW

**Review by vigneshkannan255 — APPROVED** [review]

**Comment by vigneshkannan255** [comment]
move the python file into python folder

**Review by vigneshkannan255 — CHANGES_REQUESTED** [review]
move the python file into python folder

**Comment by RashiniKaweesha** [comment]
I have move python file into python folder ane resent pull request
Thank you.

**Comment by vigneshkannan255** [comment]
remove python file which is outside the python folder and check line number 4 for spelling mistake "Enter"

File: Multiplication_Table.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# check the input whether is it a integer or not
+try:
+   # take input for the calculation value
+   num = int(input("Enete Number to multiplay : "))
```
Comment (vigneshkannan255) [inline Multiplication_Table.py:4]: here change Enete to Enter

**Review by vigneshkannan255 — CHANGES_REQUESTED** [review]

**Comment by RashiniKaweesha** [comment]
I have correct the code and delete the outside file.
Thank you.

**Pull request merged**